Sources/DolbyIORTSUIKit/Private/Views/StatsInfo/StatisticsInfoView.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  StatisticsInfoView.swift
+//
+
+import DolbyIOUIKit
+import SwiftUI
+import DolbyIORTSCore
+
+
+struct StatisticsInfoView: View {
+    private var viewModel: StatsInfoViewModel
```

Comment:
This property can be a `let` as its initialised within the initialiser and not modified later